PROCEDURES.sql

```diff
@@ -171,7 +171,7 @@ $BODY$
         IF FOUND THEN
             pID := (SELECT Participants.participantID FROM Participants WHERE Participants.participantID = intakeParticipantID);
             UPDATE Participants
-            SET Participants.race = intakeParticipantRace,
+            SET Participants.race = intakeParticipantRace AND
```

Comment:
This should still be a comma and not `AND`.

Reply:
Yea, I accidentally reverted it in my second commit. Good catch, it's fixed now!